ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventCUI-nonprod.json

```diff
@@ -562,5 +562,26 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "NOTIFY_RESPONDENT1_FOR_REQUEST_JUDGEMENT_BY_ADMISSION",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-systemupdate"
+        ],
+        "CRUD": "CRU"
+      },
+      {
+        "UserRoles": [
+          "APP-SOL-SPEC-PROFILE",
```

Comment:
Do you think we need to include citizen access profile here too?

Reply:
Better to include I think, and what will be the citizen access profile